src/init.sql

```diff
@@ -0,0 +1,202 @@
+);
+
+CREATE TABLE IF NOT EXISTS public."Venues"
+(
+    "Id" serial NOT NULL,
+    "Name" text COLLATE pg_catalog."default" NOT NULL,
+    "Description" text COLLATE pg_catalog."default",
+    "Address" text COLLATE pg_catalog."default",
+    CONSTRAINT "Venues_pkey" PRIMARY KEY ("Id")
+);
+
+CREATE TABLE IF NOT EXISTS public."Sections"
+(
+    "Id" serial NOT NULL,
+    "VenueId" integer NOT NULL,
+    "Name" text COLLATE pg_catalog."default" NOT NULL,
+    CONSTRAINT "Sections_pkey" PRIMARY KEY ("Id"),
+    CONSTRAINT "Sections_VenueId_Name_UNIQUE" UNIQUE ("VenueId", "Name")
+);
+
+CREATE TABLE IF NOT EXISTS public."Rows"
+(
+    "Id" serial NOT NULL,
+    "SectionId" integer NOT NULL,
+    "Number" smallint NOT NULL,
+    CONSTRAINT "Rows_pkey" PRIMARY KEY ("Id"),
+    CONSTRAINT "Rows_SectionId_Number_UNIQUE" UNIQUE ("SectionId", "Number")
+);
+
+CREATE TABLE IF NOT EXISTS public."Seats"
+(
+    "Id" serial NOT NULL,
+    "RowId" integer,
+    "Number" integer NOT NULL,
+    CONSTRAINT "Seats_pkey" PRIMARY KEY ("Id"),
+    CONSTRAINT "Seats_Number_RowId_UNIQUE" UNIQUE ("Number", "RowId")
+);
+
+CREATE TABLE IF NOT EXISTS public."EventSeats"
+(
+    "Id" bigserial NOT NULL,
+    "EventId" integer NOT NULL,
+    "SeatId" integer NOT NULL,
+    "PriceId" integer NOT NULL,
+    "Status" text COLLATE pg_catalog."default" NOT NULL,
+    "Version" smallint NOT NULL DEFAULT 0,
+    CONSTRAINT "EventSeats_pkey" PRIMARY KEY ("Id"),
+    CONSTRAINT "EventSeats_EventId_SeatId_UNIQUE" UNIQUE ("EventId", "SeatId")
+);
+
+CREATE TABLE IF NOT EXISTS public."Prices"
+(
+    "Id" serial NOT NULL,
+    "Type" text COLLATE pg_catalog."default" NOT NULL,
+    "Price" money NOT NULL,
+    CONSTRAINT "Prices_pkey" PRIMARY KEY ("Id")
+);
+
+CREATE TABLE IF NOT EXISTS public."CartItems"
+(
+    "Id" bigint NOT NULL,
+    "CartId" uuid NOT NULL,
+    "EventSeatId" bigint NOT NULL,
+    "PriceId" integer NOT NULL,
+    CONSTRAINT "CartItems_pkey" PRIMARY KEY ("Id"),
+    CONSTRAINT "CartItems_CartId_EventSeatId_UNIQUE" UNIQUE ("CartId", "EventSeatId")
+);
+
+CREATE TABLE IF NOT EXISTS public."Carts"
+(
+    "Id" uuid NOT NULL,
+    "UserId" integer,
+    CONSTRAINT "Carts_pkey" PRIMARY KEY ("Id")
+);
+
+CREATE TABLE IF NOT EXISTS public."Users"
+(
+    "Id" serial NOT NULL,
+    "Name" text COLLATE pg_catalog."default" NOT NULL,
+    "Email" text COLLATE pg_catalog."default" NOT NULL,
+    CONSTRAINT "Users_pkey" PRIMARY KEY ("Id"),
+    CONSTRAINT "Emain_Unique_Constraint" UNIQUE ("Email")
+);
+
+CREATE TABLE IF NOT EXISTS public."Payments"
+(
+    "Id" bigserial NOT NULL,
+    "CartId" uuid NOT NULL,
+    "Status" text COLLATE pg_catalog."default" NOT NULL,
+    CONSTRAINT "Payments_pkey" PRIMARY KEY ("Id")
+);
+
+ALTER TABLE IF EXISTS public."Events"
+    ADD CONSTRAINT "Events_VenueId_Venues_Id_FOREIGN_KEY" FOREIGN KEY ("VenueId")
+    REFERENCES public."Venues" ("Id") MATCH SIMPLE
+    ON UPDATE NO ACTION
+    ON DELETE NO ACTION
+    NOT VALID;
+
```

Comment:
Do we need this with just all the stuff being created?

Reply:
Yeah, we don't need it in initial set-up